site/src/docs/components/date-input/customisation.mdx

```diff
@@ -0,0 +1,77 @@
+---
+slug: '/components/date-input/customisation'
+title: 'DateInput - Customisation'
+---
+
+import { useState } from 'react';
+import { DateInput } from 'hds-react';
+import TabsLayout from './tabs.mdx';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Customisation
+
+### Customisation properties
+The component property `setDateClassNames` can be used to set class names for specific dates.
+
+### Customisation example
+In this example some days have different background colours to indicate that they have limited timeslots available. When days have colours to indicate something, it's important to include `legend` and `setDateAriaDescribedBy` props to inform the users of their meaning. Use the `setDateAriaDescribedBy` callback function to return corresponding LegendItem's element id for specific dates.  
+
+<Playground scope={{ DateInput, useState }}>
+
+
+  ```jsx
+import { DateInput } from 'hds-react';
+
+() => {
+    const dayHasSomeSpace = (day) => day === 12 || day === 15 || day === 24 || day === 25;
+    const dayHasLittleSpace = (day) => day === 3 || day === 4 || day === 17 || day === 19;
+
+    const setDateClassNames = (date) => {
```

Comment:
The storybook version has better example now. Too bad we have to support two sets of examples. The storybook example should be used here too, because it shows better the relations between `legend`, `setClassNames` and `setDateAriaDescribedBy`.